### PR TITLE
`Authorization`- Fix failed tests of `azurerm_marketplace_role_assignment`

### PR DESCRIPTION
**Pull request by ms-zhenhua**
## Test Result

=== RUN   TestAccRoleAssignmentMarketplace_emptyName
=== PAUSE TestAccRoleAssignmentMarketplace_emptyName
=== RUN   TestAccRoleAssignmentMarketplace_roleName
--- PASS: TestAccRoleAssignmentMarketplace_roleName (78.08s)
=== RUN   TestAccRoleAssignmentMarketplace_requiresImport
--- PASS: TestAccRoleAssignmentMarketplace_requiresImport (72.90s)
=== RUN   TestAccRoleAssignmentMarketplace_builtin
=== PAUSE TestAccRoleAssignmentMarketplace_builtin
=== RUN   TestAccRoleAssignmentMarketplace_ServicePrincipal
=== PAUSE TestAccRoleAssignmentMarketplace_ServicePrincipal
=== RUN   TestAccRoleAssignmentMarketplace_ServicePrincipalWithType
=== PAUSE TestAccRoleAssignmentMarketplace_ServicePrincipalWithType
=== RUN   TestAccRoleAssignmentMarketplace_ServicePrincipalGroup
=== PAUSE TestAccRoleAssignmentMarketplace_ServicePrincipalGroup
=== CONT  TestAccRoleAssignmentMarketplace_emptyName
=== CONT  TestAccRoleAssignmentMarketplace_ServicePrincipalWithType
=== CONT  TestAccRoleAssignmentMarketplace_ServicePrincipal
=== CONT  TestAccRoleAssignmentMarketplace_ServicePrincipalGroup
=== CONT  TestAccRoleAssignmentMarketplace_builtin
--- PASS: TestAccRoleAssignmentMarketplace_ServicePrincipalWithType (72.92s)
--- PASS: TestAccRoleAssignmentMarketplace_builtin (79.85s)
--- PASS: TestAccRoleAssignmentMarketplace_emptyName (79.94s)
--- PASS: TestAccRoleAssignmentMarketplace_ServicePrincipalGroup (90.79s)
--- PASS: TestAccRoleAssignmentMarketplace_ServicePrincipal (95.33s)
PASS
ok      github.com/hashicorp/terraform-provider-azurerm/internal/services/authorization 269.575s

### CODE REVIEW

File: internal/services/authorization/role_assignment_marketplace_resource_test.go [hunk 1]
```diff
@@ -4,6 +4,14 @@ import (
...
 	"fmt"
 	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/authorization"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/authorization/2022-04-01/roledefinitions"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/testclient"
```
Comment (WodansSon) [inline internal/services/authorization/role_assignment_marketplace_resource_test.go:15]: Is there a reason why you left and empty line between the imports?

**Review by WodansSon — CHANGES_REQUESTED** [review]
Hi @ms-zhenhua, this mostly LGTM, but I did leave a question in the PR. If you could take a look it would be much appreciated. 🚀

File: internal/services/authorization/role_assignment_marketplace_resource_test.go (hunk 1 above)
Comment (ms-zhenhua) [reply internal/services/authorization/role_assignment_marketplace_resource_test.go]: Forgot to update the import paths. Code updated.

**Comment by ms-zhenhua** [comment]
Hi @WodansSon, thank you for your review. I have resolved the comments. Kindly take another review.

**Review by WodansSon — APPROVED** [review]
@ms-zhenhua, thanks for pushing those changes. This LGTM now. 🚀

**Comment by WodansSon** [comment]
image

**Pull request merged**